recipe/patches/0015-fix-cuda-test-win.patch

```diff
@@ -7,7 +7,7 @@ index 01eeac86ae..2ded85e9a4 100644
  
  def _get_torch_cuda_version():
 -    if torch.version.cuda is None:
-+    if torch.version.cuda is None or str(torch.version.cuda) == "None":
++    if not torch.cuda.is_available() or torch.version.cuda is None or str(torch.version.cuda) == "None":
```

Comment:
I wish we'd see the call to `Generating C:/bld/libtorch_1735591958539/work/torch/version.py` and the resulting file. Can we enable the verbose build logs on CI? This looks like a kind of bug that's going to break reverse dependencies.

Reply:
I agree that this needs proper investigation - for now, I am just trying to see if some tests pass.

Reply:
Try https://github.com/isuruf/pytorch-cpu-feedstock/commit/5ed2741

Reply:
Thanks - let's see how it goes :)!